src/nodes/image.ts

```diff
@@ -10,7 +10,7 @@ import { Matrix } from 'jspdf'
 import { Viewport } from '../context/viewport'
 
 // groups: 1: mime-type (+ charset), 2: mime-type (w/o charset), 3: charset, 4: base64?, 5: body
-export const dataUriRegex = /^\s*data:(([^/,;]+\/[^/,;]+)(?:;([^,;=]+=[^,;=]+))?)?(?:;(base64))?,(?:(.|\s)*)$/i
+export const dataUriRegex = /^\s*data:(([^/,;]+\/[^/,;]+)(?:;([^,;=]+=[^,;=]+))?)?(?:;(base64))?,((?:(.|\s)*))$/i
```

Comment:
You can replace `((?:(.|\s)*))` by `((?:.|\s)*)`, should do the same and has less groups.